persistence/mapper/src/main/scala/net/liftweb/mapper/ProtoExtendedSession.scala

```diff
@@ -59,8 +59,12 @@ trait UserIdAsString {
   def userIdAsString: String
 }
 
+trait ProtoSessionCookiePath {
+  def sessionCookiePath: String = "/"
```

Comment:
Worth adding a quick scaladoc here? Also, does this need to live in its own trait?

Reply:
It does, _or_ `ContextPathExtendedCookie` would need to extend `MetaProtoExtendedSession` which semantically seemed _more_ confusing. If you disagree then it's probably worth discussing.

Can add a scaladoc here though.

Reply:
Biggest annoyance with that is that `ContextPathExtendedCookie` would need to be generic. I think this works.